samples/vs-code/covid-19.dib

```diff
@@ -10,7 +10,7 @@ Also, due to travel restrictions, you should run this at home on isolated comput
 
 Since Johns Hopkins has put COVID-19 time series data on [GitHub](https://github.com/CSSEGISandData/COVID-19), let's take a look at it. We can download it using PowerShell:
 
-#!powershell
```

Comment:
Either should be valid. Was there a reason this needed to change?

Reply:
Correct that both are valid.  I was testing round-tripping some notebooks to make sure I didn't break anything and I figured our sample .dib notebook should round-trip with no edit.